2-reply/get-updates.php

```diff
@@ -28,4 +28,10 @@
 
     // Memorizziamo il nuovo ID nel file
     file_put_contents($last_update_filename, $update_id);
+
+    $chat_id = $dati->result[0]->message->chat->id;
+    $text = $dati->result[0]->message->text;
+
+    // Rispondo al messaggio
+    http_request("https://api.telegram.org/bot{$token}/sendMessage?chat_id={$chat_id}&text={$text}");
```

Comment:
Per sicurezza, è bene passare stringhe che possono contenere caratteri non validi negli URL (spazi, caratteri speciali, etc.) attraverso la funzione `urlencode` in PHP, prima di usarle in un URL. Sarebbe quindi:

```php
http_request("https://api.telegram.org/bot{$token}/sendMessage?chat_id={$chat_id}&text=" . urlencode($text));
```

Per `$chat_id` non è strettamente necessario perché possiamo essere abbastanza certi che si tratti di un numero intero semplice.